Source/Core/Core/DSP/Interpreter/DSPIntExtOps.cpp

```diff
@@ -509,6 +512,9 @@ void nop(const UDSPInstruction opc)
 // being applied to the real registers after the main op was executed
 void applyWriteBackLog()
 {
+  using DSP::Interpreter::dsp_op_read_reg;
+  using DSP::Interpreter::dsp_op_write_reg;
```

Comment:
I was going to comment on how verbose the changes are, and if using an alias or so would make it nicer, but then I noticed this is the only place that actually does that (and just for 3 uses at that, which could be changed down to two if you use a temporary variable instead of inlining the read call):
```c++
auto value_to_write = writeBackLog[i];
#ifdef PRECISE_BACKLOG
// extended ops may change the same register; if they do the overall result is the bitwise or of both the main and extended op.
value_to_write |= DSP::Interpreter::dsp_op_read_reg(writeBackLogIdx[i]);
#endif
DSP::Interpreter::dsp_op_write_reg(writeBackLogIdx[i], value_to_write);
```

Due to that, I think I'd prefer the verbose one and get rid of those two `using`....or do you have something different in mind?